docs/src/styles/docs/layout.scss

```diff
@@ -24,7 +24,7 @@
   position: relative;
   color: var(--amplify-colors-font-tertiary);
   max-width: 60rem;
-  margin: 0 auto;
+  margin: 4rem auto 0;
```

Comment:
Can you please change this to use an `--amplify-space` CSS variable as seen on line 23?